controls/SV-204428.rb

```diff
@@ -17,25 +17,25 @@
     account required pam_faillock.so
     If the "even_deny_root" setting is not defined on both lines with the "pam_faillock.so" module, is commented out, or
```

Comment:
Since the `even_deny_root` setting needs to be present on **both** lines, I think we need to remove `describe.one` and just have two `its` statements. Also, should we be adding more to the expected pam rule to:

> Verify the operating system automatically locks the root account, for a minimum of 15 minutes, when
    three unsuccessful logon attempts in 15 minutes are made.

Reply:
Removed `describe.one`